SConstruct

```diff
@@ -412,14 +414,6 @@ if env["prereqs"]:
         if env["history"]:
             client_env.Append(CPPDEFINES = ["HAVE_HISTORY"])
 
-        if env["PLATFORM"] == 'darwin':
-            client_env.Append(FRAMEWORKS = "OpenGL")
-            client_env.AppendUnique(LIBS = ["GLEW"])
-        elif env["PLATFORM"] == 'posix':
-            opengl_config = check_output(["pkg-config", "--libs", "--cflags", "gl", "glew"])
-            opengl_flags = client_env.ParseFlags(opengl_config)
-            client_env.MergeFlags(opengl_flags)
```

Comment:
Could you rewrite history a bit to avoid this code being added and then deleted?

Reply:
Would you prefer I edit that commit, or squash the two commits together?

Reply:
I'd prefer squashing them. I think they belong together.